### DISCUSSION TITLE
How to hide a line of navigation group?

### DISCUSSION THREAD

**Discussion by Odilbukh** [filamentphp/filament#5653] [Help]
There is line on the top of the each navigation group. So how to hide the line?

![image](https://user-images.githubusercontent.com/22895615/218309920-9d38c5ee-8872-43ad-9de4-50ae050f2f58.png)

**Comment by danharrin** [comment]
CSS maybe?

**Reply by Odilbukh** [reply]
ok! But how? Where to write that css?

**Reply by danharrin** [reply]
Maybe something like `.filament-sidebar-nav li div { border: 0 !important }`

https://filamentphp.com/docs/2.x/admin/appearance#including-frontend-assets

**Reply by Odilbukh** [reply]
unfortunately it did not work
